src/api.js

```diff
@@ -1,9 +1,23 @@
+import axios from "axios";
+
 export const fetchArticles = () => {
-  return fetch("https://nc-news-portfolio.onrender.com/api/articles")
+  return axios
+    .get("https://nc-news-portfolio.onrender.com/api/articles")
     .then((response) => {
-      return response.json();
+      return response;
     })
-    .then((data) => {
+    .then(({ data }) => {
       return data.articles;
     });
 };
+
+export const fetchSingleArticle = (article_id) => {
+  return axios
+    .get(`https://nc-news-portfolio.onrender.com/api/articles/${article_id}`)
+    .then((response) => {
```

Comment:
I don't think this middle then block is necessary? it just gets the response and hands it to the next then block  (Line 17)